stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -75,6 +75,18 @@ spec:
 {{ tpl .Values.artifactory.customInitContainers . | indent 6}}
     {{- end }}
       containers:
+      {{- range .Values.artifactory.loggers }}
+      - name: {{ . | replace "_" "-" }}
+        image: busybox
+        command:
+        - tail
+        args:
+        - "-f"
+        - "/data/logs/{{ . }}.log"
```

Comment:
@k Do you mean `/var/opt/jfrog/artifactory/logs/` here?

Reply:
If that's the case replace it with `{{ .Values.artifactory.persistence.mountPath }}/logs/{{ .}}.log`